Week4/ReadingList/app/src/main/java/com/florencenjeri/readinglist/BooksFragment.kt

```diff
@@ -0,0 +1,69 @@
+package com.florencenjeri.readinglist
+
+import android.os.Bundle
+import android.view.*
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.RecyclerView
+import com.florencenjeri.readinglist.model.Books
+import com.florencenjeri.readinglist.model.BooksData
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class BooksFragment : Fragment(), BooksAdapter.BooksListClickListener {
+    val booksList = BooksData.booksRead
+    lateinit var booksRecyclerView: RecyclerView
```

Comment:
In some places you used synthetics, in some you didn't, make sure to keep it consistent!